package.json

```diff
@@ -22,7 +22,9 @@
   },
   "homepage": "https://github.com/jimkang/dce-paella-extensions",
   "devDependencies": {
-    "tape": "^3.0.3"
+    "call-next-tick": "^1.1.2",
+    "lodash": "^3.10.1",
+    "tape": "^4.2.0"
```

Comment:
That shows up in [NPM](https://www.npmjs.com/package/dce-paella-extensions). I'll update it its new location under harvard-dce.